_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -76,6 +76,34 @@ class HealthCareApp extends React.Component {
             homePhone: '555-555-5555',
             mobilePhone: '111-111-1111'
           }
+        },
+
+        // TODO: insuranceInformation should be an array where each row
+        //       contains all the following fields:
+        insuranceInformation: {
+          insuranceInfo: {
+            isCoveredByHealthInsurance: false,
+            insuranceName: '',
```

Comment:
Can we initialize these to be `undefined` if nothing specific is specified? But keep checkboxes initialized to `false`.